sentry_sdk/integrations/opentelemetry/sampler.py

```diff
@@ -68,10 +68,11 @@ def dropped_result(parent_span_context, attributes, sample_rate=None):
         else:
             reason = "sample_rate"
 
-        client.transport.record_lost_event(reason, data_category="transaction")
+        if client.transport and has_tracing_enabled(client.options):
```

Comment:
Should we explicitly check against None?

Reply:
we do this everywhere it's fine